go.mod

```
@@ -58,6 +58,7 @@ require (
 	gopkg.in/alecthomas/kingpin.v3-unstable v3.0.0-20180810215634-df19058c872c // indirect
 	gopkg.in/yaml.v2 v2.2.8
 	k8s.io/api v0.17.2
+	k8s.io/apiextensions-apiserver v0.0.0-20190918161926-8f644eb6e783
```

why not https://github.com/kubernetes/apiextensions-apiserver/releases/tag/v0.17.2 ?
```suggestion
	k8s.io/apiextensions-apiserver v0.17.2
```